Documentation/contributing/testing/scalability.rst

```diff
@@ -0,0 +1,294 @@
+
+To visualize the data, you can run Grafana locally and connect it to the Prometheus instance.
+
+.. _profiling:
+
+Accessing CPU and memory profiles
+"""""""""""""""""""""""""""""""""
+
+All of the scalability tests collect CPU and memory profiles.
+They are collected under file names like ``PodPeriodicCommand.*Profiles-stdout.*``.
+Each profile is taken periodically during the test run.
+Simplest way to visualize them is to leverage `pprof-merge <PPROF_MERGE_>`_.
+Example commands to aggregate CPU and memory profiles from whole test run:
+
+.. code-block:: bash
+
+    gsutil -m cp gs://cilium-scale-results/logs/scale-100-main/1745287079/artifacts/PodPeriodicCommand*Profiles-stdout* ./
+    for file in *.txt; do mv "$file" "${file%.txt}.tar.gz"; tar xvf "${file%.txt}.tar.gz"; done
+    pprof-merge cilium-bugtool*/cmd/pprof-cpu && mv merged.data cpu.pprof
+    pprof-merge cilium-bugtool*/cmd/pprof-heap && mv merged.data heap.pprof
+    rm -rf cilium-bugtool* PodPeriodicCommand*
+
+Then you can visualize the aggregated CPU and memory profiles by running:
+
+.. code-block:: bash
+
+    go tool pprof -http=localhost:8080 cpu.pprof
+    go tool pprof -http=localhost:8080 heap.pprof
+
+
+If you want to compare the profiles, you can compare them against the baseline extracted from different test run:
+
+.. code-block:: bash
+
+    go tool pprof -http=localhost:8080 --base=baseline_cpu.pprof cpu.pprof
+    go tool pprof -http=localhost:8080 --base=baseline_heap.pprof heap.pprof
+
+
+.. _CL2: https://github.com/kubernetes/perf-tests/tree/master/clusterloader2
+.. _CL2_GETTING_STARTED: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/docs/GETTING_STARTED.md
+.. _CL2_README: https://github.com/kubernetes/perf-tests/blob/master/clusterloader2/README.md
+.. _CLUSTERMESH_MOCK: https://github.com/cilium/scaffolding/tree/main/cmapisrv-mock
+.. _CLUSTERMESH_WORKFLOW: https://github.com/cilium/cilium/blob/main/.github/workflows/scale-test-clustermesh.yaml
+.. _EGW_MASQ_METRICS: https://github.com/cilium/cilium/blob/main/.github/actions/cl2-modules/egw/modules/masq-metrics.yaml
+.. _EGW_WORKFLOW: https://github.com/cilium/cilium/blob/main/.github/workflows/scale-test-egw.yaml
+.. _EXAMPLE_MONITOR: https://github.com/cilium/cilium/blob/main/.github/actions/cl2-modules/egw/prom-extra-podmons/podmonitor.yaml
+.. _FQDN_PERF_WORKFLOW: https://github.com/cilium/cilium/blob/main/.github/workflows/fqdn-perf.yaml
+.. _GSUTIL_INSTALL: https://cloud.google.com/storage/docs/gsutil_install
+.. _NETPOL_SCALE_TEST: https://github.com/cilium/cilium/tree/main/.github/actions/cl2-modules/netpol
+.. _PERFDASH: https://github.com/cilium/scaffolding/tree/main/scale-tests
+.. _PPROF_MERGE: https://github.com/rakyll/pprof-merge
+.. _SCALE_100_WORKFLOW: https://github.com/cilium/cilium/blob/main/.github/workflows/scale-test-100-gce.yaml
+.. _SLACK_CHANNEL: https://slack.cilium.io
+.. _TEST_RUN: https://github.com/cilium/cilium/actions/runs/14584355333
```

Comment:
Please find another link here. This is a single workflow run, and the results and logs are not persistent over long durations (we keep them, what, three months?), which means that it will break in a few months at most.

You could point to the workflow runs list: https://github.com/cilium/cilium/actions/workflows/scale-test-100-gce.yaml and explain how do display results from there.